ckanext/qdes_schema/fanstatic/accordion.js

```diff
@@ -0,0 +1,12 @@
+jQuery(document).ready(function () {
+      
+  document.addEventListener('keyup', function (e) {
+    console.log(e.key);
+    if (e.key === 'Enter' || e.key === ' ' || e.key === 'Spacebar') {
+      var checked = document.activeElement.parentNode.querySelector("input[type]").checked;
```

Comment:
I think once we have a better selected we should be able to get the element from the `e` event object and toogle the checked value.